Classes/Form/Field/Text.php

```diff
@@ -173,4 +182,41 @@ public function setFormat($format)
     {
         $this->format = $format;
     }
+
+    /**
+     * @return string
+     */
+    public function getRichtextConfiguration()
+    {
+        return $this->richtextConfiguration;
```

Comment:
Although it makes the overall experience a little less predictable, I think we should move the body of computeRichtextConfiguration() to this getter and drop computeRichtextConfiguration (along the lines of getItems() on AbstractMultiValueFormField). Then you always get the applied configuration also when you access the value separately from buildConfiguration(), like you do with select items.

Caching the resolved configuration in the internal property is completely okay and should help performance, e.g. you can call buildConfiguration() over and over without re-analysing the TS parameter.